src/json-patch-duplex.js

```diff
@@ -268,10 +267,10 @@ var jsonpatch;
         if (callback) {
             observer.callback = callback;
             observer.next = null;
-            var dirtyCheck = function () {
+            var dirtyCheck = () => {
```

Comment:
As above, in this branch we would like to keep IE11 support for quite long. We can drop it for `Proxy`-observer. But I feel arrow functions are not worth loosing x-platform support.